docs/source/gsoc.rst

```diff
@@ -0,0 +1,57 @@
+GSoC 2019 ideas
+===============
+
+Why LiberTEM
+--------------
+
+`LiberTEM <https://libertem.github.io/LiberTEM/>`_ LiberTEM is an open source platform for high-throughput distributed processing of pixelated scanning transmission electron microscopy (STEM) data. It is created to deal with the terabytes of data that modern high-speed high-resolution detectors for electron microscopy can produce. 
+
```

Comment:
may be add an image here

Reply:
@ReimarBauer I added two in f848113 -- what do you think?